app/views/unattended/provisioning_templates/discovery/debian_kexec.erb

```diff
@@ -40,6 +40,6 @@ require:
 {
 "kernel": "<%= @kernel_uri %>",
 "initram": "<%= @initrd_uri %>",
-"append": "url=<%= foreman_url('provision') + "&static=yes" %> interface=<%= mac %> netcfg/get_ipaddress=<%= ip %> netcfg/get_netmask=<%= mask %> netcfg/get_gateway=<%= gw %> netcfg/get_nameservers=<%= dns %> netcfg/disable_dhcp=true netcfg/get_hostname=<%= @host.name %> BOOTIF=<%= bootif %> <%= options.compact.join(' ') %>",
+"append": "url=<%= foreman_url('provision', { static: 'yes' })%> interface=<%= mac %> netcfg/get_ipaddress=<%= ip %> netcfg/get_netmask=<%= mask %> netcfg/get_gateway=<%= gw %> netcfg/get_nameservers=<%= dns %> netcfg/disable_dhcp=true netcfg/get_hostname=<%= @host.name %> BOOTIF=<%= bootif %> <%= options.compact.join(' ') %>",
```

Comment:
This looks like an unrelated change. Your commit has 3d600a5631927f0c9b485cd0527cc6febf0356ba as a parent while 3305096ae310c9851fc2e17591f9a2579d80db8d is the current latest in develop, which should include the changes. I suspect this is something that went wrong in a rebase.

Please rebase on the latest develop.

Reply:
Looks like that didn't make a change. In https://community.theforeman.org/t/git-remote-fetch-pull-and-rebase/29949 I've written up some tips on rebasing.

Reply:
You mean a link? Because including the content there is wrong. There are many places which aren't Foreman, but are just plain git. Smart Proxy or foreman-documentation for example.

Reply:
I meant to add it as a link to the document, if think it doesn't belong there we can skip it.